app/src/main/java/com/github/se/eventradar/ui/qrCode/QrCodeFriendUi.kt

```diff
@@ -0,0 +1,165 @@
+        modifier =
+            Modifier.fillMaxWidth()
+                .padding(top = 8.dp)
+                .constrainAs(tabs) {
+                  top.linkTo(logo.bottom, margin = 16.dp)
+                  start.linkTo(parent.start)
+                  end.linkTo(parent.end)
+                }
+                .testTag("tabs"),
+        contentColor = MaterialTheme.colorScheme.primary) {
+          Tab(
+              selected = selectedTabIndex == 0,
+              onClick = { selectedTabIndex = 0 },
+              modifier = Modifier.testTag("My QR Code"),
+          ) {
+            Text(
+                text = "My QR Code",
+                style =
+                    TextStyle(
+                        fontSize = 19.sp,
+                        lineHeight = 17.sp,
+                        fontFamily = FontFamily(Font(R.font.roboto)),
+                        fontWeight = FontWeight(500),
+                        color = MaterialTheme.colorScheme.onPrimaryContainer,
+                        textAlign = TextAlign.Center,
+                        letterSpacing = 0.25.sp,
+                    ),
+                modifier = Modifier.padding(bottom = 8.dp))
+          }
+          Tab(
+              selected = selectedTabIndex == 1,
+              onClick = { selectedTabIndex = 1 },
+              modifier = Modifier.testTag("Scan QR Code")) {
+                Text(
+                    text = "Scan QR Code",
+                    style =
+                        TextStyle(
+                            fontSize = 19.sp,
+                            lineHeight = 17.sp,
+                            fontFamily = FontFamily(Font(R.font.roboto)),
+                            fontWeight = FontWeight(500),
+                            color = MaterialTheme.colorScheme.onPrimaryContainer,
+                            textAlign = TextAlign.Center,
+                            letterSpacing = 0.25.sp,
+                        ),
+                    modifier = Modifier.padding(bottom = 8.dp))
+              }
+        }
+
+    if (selectedTabIndex == 0) {
+      Toast.makeText(context, "My Qr Code not yet available", Toast.LENGTH_SHORT).show()
+    } else {
+      Column(modifier = Modifier.testTag("QrScanner")) { QrCodeCamera().QrCodeScanner() }
+    }
+    BottomNavigationMenu(
+        onTabSelected = { tab -> navigationActions.navigateTo(tab) },
+        tabList = TOP_LEVEL_DESTINATIONS,
+        selectedItem = TOP_LEVEL_DESTINATIONS[0],
+        modifier =
+            Modifier.testTag("bottomNavMenu").constrainAs(bottomNav) {
+              bottom.linkTo(parent.bottom)
+              start.linkTo(parent.start)
+              end.linkTo(parent.end)
+            })
+  }
+}
+
+// private val dummyQrCodeScanned: (String) -> Unit = { qrCode ->
+//  Log.d("QRCodeScanner", "QR Code Scanned: $qrCode")
+//  // You can perform any additional logic here for testing
+// }
+
+// @androidx.compose.ui.tooling.preview.Preview
```

Comment:
You can leave this uncommented and add the annotation described in #180 (once it's approved)